### PR TITLE
Fixed readmes, updated deps and pulled in some changes from forks.

### PR DESCRIPTION
**Pull request by niemyjski**
I'm in the process of testing these changes as we speak but I wanted to get the pr for this rolling 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
-Uses OpenCV and Tensorflow to perform realtime object detection locally for IP cameras. Designed for integration with HomeAssistant or others via MQTT.
+Uses OpenCV and Tensorflow to perform realtime object detection locally for IP
+cameras. Designed for integration with HomeAssistant or others via MQTT.
 
-![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+![Supports aarch64 Architecture][aarch64-shield]
+![Supports amd64 Architecture][amd64-shield]
+![Supports armhf Architecture][armhf-shield]
+![Supports armv7 Architecture][armv7-shield]
+![Supports i386 Architecture][i386-shield]
```
Comment (MauriceKok) [inline README.md]: Maybe I'm seeing this wrong. But doens't Blake Blackshear only provide a docker image for linux/amd64 ?

File: frigate-beta/README.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+![Supports armhf Architecture][armhf-shield]
+![Supports armv7 Architecture][armv7-shield]
+![Supports i386 Architecture][i386-shield]
 
-This repository contains two add-ons:  
-frigate: uses the current production release 0.5.0  
-frigate-beta: uses the current beta release 0.4.0-beta
+This repository contains two add-ons:
+frigate: uses the current production release 0.5.0
+frigate-beta: uses the current beta release 0.5.4-rc4
```
Comment (MauriceKok) [inline frigate-beta/README.md:12]: I think I would like to follow the release channels of the original project, so prod beta and rc

**Comment by mories76** [comment]
Thanks for the PR, awaiting you reactions before merging

File: README.md (hunk 1 above)
Comment (niemyjski) [reply README.md]: Yes, but this was in the original readme, I just formatted it... I removed the extras.

File: frigate-beta/README.md (hunk 2 above)
Comment (niemyjski) [reply frigate-beta/README.md:12]: Would be nice to figure out if we could just specify configuration setting for a docker tag and then just have one.

**Comment by niemyjski** [comment]
I commented, would it be possible to get write access to help out on this project where I can? Otherwise I can just do pr's :-)

**Comment by mories76** [comment]
Invite pending, the floor is yours.
I never guessed I would receive a PR ;-)
Isn't this was opensource is all about

**Comment by niemyjski** [comment]
Thanks! If you are good with me merging this I can, I'll create pr's and ask for your review for any future tasks :) and merge any bug fixes in right away.

**Comment by mories76** [comment]
Just merged it, thx for your input

**Pull request merged**